src/main/proto/wfa/measurement/api/v2alpha/crypto.proto

```diff
@@ -41,3 +41,46 @@ message SignedData {
   // Cryptographic digital signature of `data`.
   bytes signature = 2;
 }
+
+// A public key for asymmetric encryption.
+message EncryptionPublicKey {
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+
+    // Elliptic curve key on the NIST P-256 curve.
+    //
+    // Algorithm identifier:
+    //   id-ecPublicKey
+    //   prime256v1
+    EC_P256 = 1;
+  }
+  // Type of encryption key. Required.
+  Type type = 1;
+
+  // The ASN.1 SubjectPublicKeyInfo in DER format. Required.
+  //
+  // The AlgorithmIdentifier must match `key_type`.
+  bytes public_key_info = 2;
+}
+
+// Description of a cipher suite for hybrid encryption using the KEM/DEM
+// paradigm.
+message HybridCipherSuite {
+  enum KeyEncapsulationMechanism {
+    KEY_ENCAPSULATION_MECHANISM_UNSPECIFIED = 0;
+
+    // ECDH over NIST P-256 with HKDF using HMAC-SHA256.
+    ECDH_P256_HKDF_HMAC_SHA256 = 1;
```

Comment:
Are we wanting to include all this in the proto?

Reply:
I think something went wrong in merging -- this definitely shouldn't be part of this PR. This is from a PR that Sanjay already landed.